mixins/visual/responsive-visual.vue

```diff
@@ -19,6 +19,29 @@ export default
 		portraitVideo: Object
 	}
 
+	head: ->
+		return {} unless @isResponsiveImage and @preload
```

Comment:
It's ok to return void from this

```suggestion
		return unless @isResponsiveImage and @preload
```